plugins/home/src/homePageComponents/FeaturedDocs/FeaturedDocs.tsx

```diff
@@ -0,0 +1,154 @@
+import { catalogApiRef, CatalogApi } from '@backstage/plugin-catalog-react';
+import { useApi } from '@backstage/core-plugin-api';
+import { EntityFilterQuery } from '@backstage/catalog-client';
+import { ClassNameMap } from '@material-ui/styles';
+
+import { makeStyles, Theme, Typography } from '@material-ui/core';
+
+type DocsCardProps = {
+  filter: EntityFilterQuery;
+  color?: 'inherit' | 'primary' | 'secondary' | undefined;
+  customStyles?: ClassNameMap<string> | undefined;
+  emptyState?: React.ReactNode | undefined;
+  subLinkText?: string | undefined;
+  title?: React.ReactNode | string | undefined;
+};
+
+const useStyles = makeStyles<Theme>(() => ({
+  docDescription: {
+    fontSize: 16,
+    fontWeight: 400,
+    marginBottom: '16px',
+    marginTop: '12px',
+  },
+  docSubLink: {
+    fontSize: 12,
+    fontWeight: 700,
+    lineHeight: 2,
+  },
+  docsTitleLink: {
+    fontSize: 16,
+    fontWeight: 600,
+    lineHeight: 2,
+  },
+}));
+
+/**
+ * A component to display specific Featured Docs.
+ * @param {EntityFilterQuery} filter - The entity filter used to display only the intended item/s
+ * @param {'inherit' | 'primary' | 'secondary' | undefined} [color] - An optional color which can be customized through themes
+ * @param {ClassNameMap<string> | undefined} [customStyles] - An optional ClassNameMap created with makeStyles
+ * @param {React.ReactNode | undefined} [emptyState] - An optional ReactNode for empty states
+ * @param {string | undefined} [subLinkText] - An optional string to customize sublink text
+ * @param {React.ReactNode | string | undefined} [title] - An optional string or ReactNode to customize the card title
+ *
+ * @public
+ */
+export const FeaturedDocs = (props: DocsCardProps) => {
+  const { color, customStyles, emptyState, filter, subLinkText, title } = props;
+  const linkText = subLinkText || 'LEARN MORE';
+  const defaultStyles = useStyles();
+  const styles = customStyles || defaultStyles;
+  const catalogApi: CatalogApi = useApi(catalogApiRef);
+  const {
+    value: entities,
+    loading,
+    error,
+  } = useAsync(async () => {
+    const response = await catalogApi.getEntities({
+      filter: filter,
+    });
+    return response.items;
+  });
+
+  if (loading) {
+    return <Progress />;
+  }
+
+  if (error) {
+    return (
+      <WarningPanel
+        severity="error"
+        title="Could not load available documentation."
+      >
+        <CodeSnippet language="text" text={error.toString()} />
+      </WarningPanel>
+    );
+  }
+
+  return (
+    <InfoCard variant="gridItem" title={title || 'Featured Docs'}>
+      {entities?.length
+        ? entities.map(d => (
+            <div key={d.metadata.name} data-testid="docs-card-content">
+              <Link
+                className={styles.docsTitleLink}
+                data-testid="docs-card-title"
+                color={color || 'primary'}
+                to={`/docs/${d.metadata.namespace || 'default'}/${d.kind}/${
```

Comment:
These are all hardcoded routes and we don't know for sure that that's where the docs pages are mounted. Should it instead resolve rootDocsRouteRef? https://github.com/backstage/backstage/blob/825fcd3f357fd9c9a02d9de19a60d775ff27caa4/plugins/techdocs/src/routes.ts#L23

hmmmmm i see it's in a plugin package though instead of in the `-react` package so you should not import that ... ok let's leave this as a future improvement maybe :/

Reply:
I added a path parameter as a backup to this.

Reply:
Thanks for the review @freben, just pushed the requested changes 👍